WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewModelProtocol.swift

```diff
@@ -84,6 +84,10 @@ protocol ProductFormViewModelProtocol {
 
     func updateStatus(_ isEnabled: Bool)
 
+    func updateDownloadables(downloads: [ProductDownload]?,
+                             downloadLimit: Int64?,
+                             downloadExpiry: Int64?)
```

Comment:
Is there a reason why they are declared as optional?

Reply:
I was not sure about the difult values at that moment. And the behaviors on the absence of value. A point to discuss.

Reply:
Yeah, I think you are right. Those should be non-optional since we will be getting some default value for sure. Changed and tested it.